apps/patient/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "chakra-react-select": "^4.6.0",
     "graphql-request": "^5.2.0",
+    "moment": "^2.29.4",
```

Comment:
Should we choose another library? From their readme:

```
Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library.
```

Reply:
Sure, I'll explore a bit